src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -56,13 +102,31 @@ const propTypes = {
 
     /** Forwarded ref to FloatingActionButtonAndPopover */
     innerRef: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
+
+    /** Information on the last taken action to display as Quick Action */
+    quickAction: PropTypes.shape({
+        action: PropTypes.string,
+        chatReportID: PropTypes.string,
+        targetAccountID: PropTypes.number,
+        isFirstQuickAction: PropTypes.bool,
```

Comment:
I don't see where we are using this, am I missing something?

Reply:
We'll be using that to determine if we show an educational tooltip about the quick action: https://github.com/Expensify/App/issues/38054

Reply:
Yep. Not in use yet, but it'll come with the follow-up PRs